Marlin/src/gcode/control/M3-M5.cpp

```diff
@@ -78,6 +81,10 @@ void GcodeSuite::M3_M4(const bool is_M4) {
     return cutter.unitPower;
   };
 
+  #if ENABLED(SPINDLE_SERVO)
+    servo[SPINDLE_SERVO_ID].move(get_s_power());
```

Comment:
Which `CUTTER_POWER_UNIT` are you expecting people to use? I don't see anything in the `get_s_power` code path that would convert thing to an angle.